### PR TITLE
External-dns improve azure support

### PR DESCRIPTION
**Pull request by duga3**
* Automatically mount the azure configuration as VolumeMount when using azure as provider.
* Bump external chart to latest stable version (0.4.8)
* Make RBAC api version configurable

### CODE REVIEW

**Comment by duga3** [comment]
#3389 should go in first

File: stable/external-dns/values.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ sources:
 publishInternalServices: false
 
 ## The DNS provider where the DNS records will be created (options: aws, google, inmemory, azure )
-provider: aws
+provider: azure
```
Comment (unguiculus) [inline stable/external-dns/values.yaml]: Why change the default?

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Review by unguiculus — APPROVED** [review]

**Comment by unguiculus** [comment]
cc @rk295

**Pull request merged**